### PR TITLE
Add /bin to sudo secure_path

### PR DESCRIPTION
**Pull request by nbonnotte**
see issue #524

cf. PR #525

### CODE REVIEW

**Comment by parente** [comment]
@nbonnotte Thanks for the PR. It looks like the tests failed waiting for a notebook server to startup in many cases. I'll retry them, but it's possible the change in start.sh broke something that needs local debugging.

File: base-notebook/test/test_container_options.py [hunk 1]
```diff
@@ -77,7 +77,6 @@ def test_sudo_path(container):
     c = container.run(
         tty=True,
         user='root',
-        environment=['GRANT_SUDO=yes'],
```
Comment (parente) [inline base-notebook/test/test_container_options.py]: It'd be nice to keep the `GRANT_SUDO` option in this test to check that `jovyan` can run the `sudo which jupyter` command, and write another test (or parameterize this one) to check the case where the container launches as root without granting the user sudo.

**Comment by parente** [comment]
Replicating the test that is hanging:

```
~/p/docker-stacks ❯❯❯ docker run -it --rm --user root jupyter/base-notebook start.sh sudo which jupyter
Set username to: jovyan
usermod: no changes
Executing the command: sudo which jupyter
[sudo] password for jovyan:
```

The issue is that `start.sh` does `exec sudo -E -H -u $NB_USER PATH=$PATH $cmd` to switch to the `$NB_USER` which is `jovyan` by default. But then as `jovyan`, it's trying to execute the `sudo which jupyter` command given in the test or on the command line above. 

When running the container as `--user root` with the `start.sh` script, the sudo test is implicit. I think we just need the new test to check `which jupyter` instead of `sudo which jupyter`. That or the test should not use `start.sh` and just run `sudo which jupyter` directly.

**Comment by nbonnotte** [comment]
@parente The tests are now passing

**Pull request merged**